### DISCUSSION TITLE
Caddyfile parser

### DISCUSSION THREAD

**Discussion by poperigby** [tree-sitter/tree-sitter#3557] [Grammars]
It looks like somebody has already started one, but it seems to be abandoned: https://github.com/Samonitari/tree-sitter-caddy

**Comment by amaanq** [comment]
Please don't file parser requests in discussions - it'd be nice if there were a Caddyfile parser, but if we were to allow requests here then anyone and everyone would be filing requests, polluting discussions.

**Reply by poperigby** [reply]
Sorry, I wasn't aware that was a rule.

**Reply by amaanq** [reply]
No worries at all! Coming from nvim-treesitter we had a *ton* of parser requests [here](https://github.com/nvim-treesitter/nvim-treesitter/issues/2282) to the point where it became unfeasible to keep open, and we got rid of it. Sorry if I came off as harsh - maybe a notice when opening a discussion like "parser requests won't be accepted" would be a nice thing to add.

**Reply by poperigby** [reply]
It's okay, you didn't come off as harsh. Yeah, that would be a good idea I think.
